pyvista/plotting/utilities/sphinx_gallery.py

```diff
@@ -66,9 +66,9 @@ class Scraper:
 
     """
 
-    def __repr__(self):  # pragma: no cover
-        """Return a stable representation of the class."""
-        return f"<{self.__class__.__name__}"
+    def __repr__(self):
+        """Return a stable representation of the class instance."""
+        return f"<{self.__class__.__name__} object>"
```

Comment:
Actually, I'm thinking this could be
```suggestion
        return f"<{type(self).__name__} object>"
```
Thoughts?

Reply:
I guess we can and more safe.